WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -428,7 +428,7 @@ public void onPostButtonClicked(int buttonType, PostsListPost post) {
      * send the passed post to the trash with undo
      */
     private void trashPost(final PostsListPost post) {
-        if (!isAdded() || !NetworkUtils.checkConnection(getActivity())) {
+        if (!isAdded() || (!post.isLocalDraft() && !NetworkUtils.checkConnection(getActivity()))) {
```

Comment:
This would benefit from a comment explaining why we don't need to check connectivity for local drafts.